app/src/main/java/com/mapbox/maps/testapp/examples/DebugModeActivity.kt

```diff
@@ -172,12 +171,6 @@ class DebugModeActivity : AppCompatActivity() {
     }
   }
 
-  override fun onDestroy() {
-    super.onDestroy()
-    mapboxMap.unsubscribe(observable)
-    mapboxMap.unsubscribeResourceRequest(extensionObservable)
-  }
-
```

Comment:
Could you explain the reason for this change, changes like these are great to highlight in your PR description.

Reply:
Observable will be automatically unsubscribed in `MapboxMap.onDestroy()`